src/olympia/api/paginator.py

```diff
@@ -49,3 +52,36 @@ def page(self, number):
                 raise EmptyPage('That page contains no results')
 
         return page
+
+
+class CustomPageNumberPagination(PageNumberPagination):
+    """Custom pagination implementation to hook in our `ESPaginator`."""
+    page_size_query_param = 'page_size'
```

Comment:
I set `max_page_size` but left `last_page_strings` untouched - I think it's useful to implement a proper paginator for the API consumer.